product/reports/180_Configuration Management - Instances/010 Amazon - Active VMs.yaml

```diff
@@ -0,0 +1,84 @@
+---
+title: Amazon - Active VMs
+rpt_group: Custom
+rpt_type: Custom
+priority: 20
+db: VmCloud
```

Comment:
as this model `VmCloud` is not defined in 
https://github.com/ManageIQ/manageiq/blob/master/config/miq_expression.yml#L2
please add it to this yaml.
this is will ensure that we can create the report  based on this model:
![Uploading Screen Shot 2017-06-26 at 21.22.48.png…]()